js/ui/toast/ui.toast.js

```diff
@@ -91,81 +90,55 @@ const Toast = Overlay.inherit({
+                type: 'fade',
+                duration: 200,
+                from: 1,
+                to: 0
+            }
+        };
+
         return this.callBase().concat([
             {
-                device: { platform: 'android' },
+                device(device) {
+                    return device.deviceType === 'phone';
+                },
                 options: {
+                    width: `calc(100vw - ${DEFAULT_MARGIN * 2}px)`,
+                    displayTime: isMaterial() ? 4000 : 2000,
                     hideOnOutsideClick: true,
-
-                    width: 'auto',
-
-                    position: {
-                        at: 'bottom left',
-                        my: 'bottom left',
-                        offset: '20 -20'
-                    },
-
-                    animation: {
-                        show: {
-                            type: 'slide',
-                            duration: 200,
-                            from: {
-                                position: {
-                                    my: 'top',
-                                    at: 'bottom',
-                                    of: window
-                                }
-                            },
-                        },
-                        hide: {
-                            type: 'slide',
-                            duration: 200,
-                            to: {
-                                position: {
-                                    my: 'top',
-                                    at: 'bottom',
-                                    of: window
-                                }
-                            },
-                        }
-                    }
-                }
-            },
-            {
-                device: function(device) {
-                    const isPhone = device.deviceType === 'phone';
-                    const isAndroid = device.platform === 'android';
-
-                    return isPhone && isAndroid;
+                    animation: tabletAndMobileAnimation,
                 },
-                options: {
-                    width: '100vw',
-
-                    position: {
-                        at: 'bottom center',
-                        my: 'bottom center',
-                        offset: '0 0'
-                    }
-                }
             },
             {
-                device: function(device) {
-                    return device.deviceType === 'phone';
+                device(device) {
+                    return device.deviceType === 'tablet';
                 },
                 options: {
-                    width: '100vw',
-                }
+                    width: 'auto',
+                    maxWidth: `calc(100vw - ${DEFAULT_MARGIN * 2}px)`,
+                    displayTime: isMaterial() ? 4000 : 2000,
+                    hideOnOutsideClick: true,
+                    animation: tabletAndMobileAnimation,
+                },
```

Comment:
all this option seem to be the same as for phones. Can we unify it?
```javascript
device({ deviceType }) {
    return deviceType === 'phone' || deviceType === 'tablet';
},
options: {
      width: 'auto',
      maxWidth: `calc(100vw - ${DEFAULT_MARGIN * 2}px)`,
      displayTime: isMaterial() ? 4000 : 2000,
      hideOnOutsideClick: true,
      animation: tabletAndMobileAnimation,
  },
```

Reply:
Not all